Content.Server/Goobstation/Changeling/ChangelingSystem.cs

```diff
@@ -342,7 +342,16 @@ public bool TryToggleItem(EntityUid uid, ProtoId<EntityPrototype> proto, ref Ent
 
         return true;
     }
-
+    public bool DNAHistory(EntityUid uid, ChangelingComponent comp, TransformData data)//AbsorbedDNAHistory
```

Comment:
i don't see it being used anywhere. why add it?

Reply:
used in AddDNA(uid, comp, data, DNAHistory(uid, comp, data)); under TryStealDNA() to check if it shud award a objective point